src/pages/commits/ActiveCommits/utils.ts

```diff
@@ -41,3 +48,31 @@ export const getAllTaskStatsGroupedByColor = (
 
   return idToGroupedTaskStats;
 };
+
+// Used in Commit Chart Component to calculate bar heights
+export function calculateBarHeight(
+  value: number,
+  max: number,
+  total: number,
+  chartType: string
+) {
+  if (chartType === ChartTypes.Percentage) {
+    return `${(value / total) * 100}%`;
+  }
+  return `${(value / max) * 100}%`;
+}
+
+// Find zero count statuses for commit chart tooltip
+export function getZeroCountStatus(currColors: ColorCount[]) {
+  const existingStatusSet = new Set(
+    currColors.map(({ umbrellaStatus }) => umbrellaStatus)
+  );
+  const umbrellaStatuses = Array.from(
+    new Set(
+      Object.values(mapTaskStatusToUmbrellaStatus).map((status) => status)
+    )
+  );
+  return umbrellaStatuses.filter(
+    (umbrellaStatus) => !existingStatusSet.has(umbrellaStatus)
+  );
+}
```

Comment:
I was able to clean this up a little bit. 
```suggestion
export const getStatusesWithZeroCount = (colors: ColorCount[]) => {
  const availableStatuses = colors.map(({ umbrellaStatus }) => umbrellaStatus);
  const allStatuses = Object.values(mapTaskStatusToUmbrellaStatus);
  return Array.from(
    new Set(allStatuses.filter((status) => !availableStatuses.includes(status)))
  );
};

```